pkg/integration/trivy/analyzer.go

```diff
@@ -49,16 +49,21 @@ func (TrivyAnalyzer) analyzeVulnerabilityReports(a common.Analyzer) ([]common.Re
 
 		// For each pod there may be multiple vulnerabilities
 		var failures []common.Failure
+		distinctFailures := make(map[string]common.Failure)
```

Comment:
@michael12312 do you know why Trivy can report some duplicate CVEs for a specified pod? Is it a bug in Trivy?

Reply:
hi @gyliu513 

Per my testing, Trivy truely report duplicate CVEs. So for fixing, I made this PR.

Can you please help to do the code review and approval? Thanks very much!